infra/helper.py

```diff
@@ -713,10 +716,23 @@ def build_fuzzers_impl(  # pylint: disable=too-many-arguments,too-many-locals,to
 
 def build_fuzzers(args):
   """Builds fuzzers."""
+  if args.engine == 'centipede':
+    # A hack to place target binary (with san) into a subdir of project.
```

Comment:
Add more context here in the comment: Centipede requires separate binaries for sanitizers.